package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Covid Stats in Telegram",
   "main": "index.js",
   "scripts": {
-    "start": "node app/index.js",
+    "start": "nodemon app/index.js",
```

Comment:
Thanks, We needed this for development. But I don't want to use `nodemon` in the production environment and `start` command is used in the server to start the app. Can you create a new command like
"dev": "nodemon app/index.js" ?

Reply:
Oh yeah, sure.

Reply:
Done ✅